### PR TITLE
 Improvements to code base

### PR DESCRIPTION
**Pull request by TheItachiUchiha**
This PR has multiple code improvements. As I am new to Kotlin, there might be a chance that I am doing something wrong here. Please feel free to comment.

### CODE REVIEW

File: src/main/kotlin/com/example/demo/views/TileGUI.kt [hunk 1]
```diff
@@ -83,13 +85,14 @@ class TileGUI : Fragment() {
...
                                         it
-                                    }
-                                    graphic.setOnMouseEntered {
-                                        graphic.addClass(highlightTile)
-                                    }
-
-                                    graphic.setOnMouseExited {
-                                        graphic.removeClass(highlightTile)
+                                    }.apply {
+                                        setOnMouseEntered {
```
Comment (ahinchman1) [inline src/main/kotlin/com/example/demo/views/TileGUI.kt:89]: I believe this is the preferred way of adding event filters to particular nodes, however, I am not entirely sure if there's a benefit to this over the way I had previously.  I personally think it's a little bit harder to read, but I think it does fit the style of kotlin.  I think this is something I'd ask about on the #general channel and tag you in it

Comment (ahinchman1) [reply src/main/kotlin/com/example/demo/views/TileGUI.kt:89]: I'm accepting this, now that I don't have a cache in my graphic nodes anymore

**Review by ahinchman1 — APPROVED** [review]
Nice work on the syntax cleanup.

File: src/main/kotlin/com/example/demo/views/TileGUI.kt (hunk 1 above)
Comment (TheItachiUchiha) [reply src/main/kotlin/com/example/demo/views/TileGUI.kt:89]: There is no need for cache, but if you still want it, you can use `cellCache` in data grid.

**Pull request merged**